lineman/app/models/comment_model.coffee

```diff
@@ -30,6 +30,12 @@ angular.module('loomioApp').factory 'CommentModel', (BaseModel, AppConfig) ->
     isReply: ->
       @parentId?
 
+    parent: ->
+      @recordStore.comments.find(@parentId)
```

Comment:
parentAuthorName is actually serialized out on the comment now, because we can't guarantee having the parent (what if you reply to a comment from 10 years and 500 comments ago?) 

So, shouldn't need this.